x/burner/keeper/burn_test.go

```diff
@@ -0,0 +1,54 @@
+package keeper_test
+
+import (
+	"testing"
+
+	keepertest "github.com/elys-network/elys/testutil/keeper"
+	"github.com/elys-network/elys/x/burner/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestShouldBurnTokens(t *testing.T) {
+	k, ctx := keepertest.BurnerKeeper(t)
+
+	params := types.Params{
+		EpochIdentifier: "test_epoch",
+	}
+	k.SetParams(ctx, params)
+
+	require.True(t, k.ShouldBurnTokens(ctx, "test_epoch"))
+	require.False(t, k.ShouldBurnTokens(ctx, "invalid_epoch"))
+}
+
+// func TestBurnTokensForAllDenoms(t *testing.T) {
```

Comment:
I think this test could be removed if not required or just enable it in case it's needed

Reply:
test fixed